### ISSUE TITLE
Organize data directory on Dahlquist Lab repository; use README.md to make table of contents

### ISSUE THREAD

**Issue by kdahlquist** [kdahlquist/GRNmap#336]
@kdahlquist to organize data directory on Dahlquist Lab repository

**Comment by kdahlquist** [comment]
@kdahlquist will organize the DahlquistLab repository so that we have clear directories for storing both "raw" and "processed" data, as well as a PowerPoint with graphs and a Word document for discussion of the data #336.

* Both the PowerPoint and Word document should provide hyperlinks to the actual data files on each slide/discussion paragraph
* The README.md file in each directory will serve as a table of contents for the directory, listing each file and its contents

**Comment by kdahlquist** [comment]
I made a first pass at organizing the DahlquistLab data directory.  I moved some files around, renamed some files, and created README.md files at each level of the directory.  The individual README.md files contain short comments and questions that need to be resolved.  We will talk about this at the meeting on Thursday, but it would be helpful if each team member took a look beforehand and started uploading or moving files.

To edit the README.md file, click on it and then choose the pencil icon.  We are going to build a table of contents for each directory and notes in the README files.
